shared/src/main/scala/amf/client/model/domain/Shape.scala

```diff
@@ -47,8 +47,8 @@ trait Shape extends DomainElement with Linkable with NamedDomainElement {
     this
   }
 
-  def withDefaultValue(default: DataNode): this.type = {
```

Comment:
`default` is a typescript keyword. This is tricky because if someone is using Scala that has by argument-by-name it breaks backwards compatibility. Really sure no one uses this though.